_posts/release/2020-02-29-accumulo-1.10.0.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Accumulo 1.10.0 (Draft)  
+---
+
+28 August 2020
+
+Apache Accumulo 1.10 is a maintenance release of the 1.9.3 version with additional internal 
+improvements.  This release raises the Java language level for the source code 
+from 1.7 to 8 and requires a Java runtime environment of Java 8 or later.  The source code level 
+change aligns the Accumulo code base Java language version requirements with the 2.0 branches and 
+allows Java language enhancements such as lambda expressions and improved interface features to be 
+used in all active branches of the Accumulo code base.
+ 
+With the adoption of [semver](https://semver.org/spec/v2.0.0.html), the minor number version change
+(from 1.9 to 1.10) signals this release does not modify the Accumulo public API, but that the 
+release contains internal improvements beyond changes that are permitted by semver in a patch 
+release. This release contains contributions from more than 13 contributors from the Apace Accumulo
+community in over 80 commits and 16 months of work since the 1.9.3 release. The following release 
+notes highlight some of the changes. If anything is missing from this list, please
+[contact](https://accumulo.apache.org/contact-us) the developers to have it included.
+
+## Long Term Support (LTS)
+
+The Apache Accumulo community has adopted a formal Long Term Support (LTS) release policy.  Accumulo 
+version 1.10 is the first LTS version and is expected to be the last major / minor release of the 
+1.x line. With the LTS designation, 1.10.0 will receive critical bug and security fixes for the 
+defined support period.  No new features will be added to the 1.10 line and all new feature 
+development will occur on the 2.x line.
+
+| Version | Release Date | Maintenance Period |
+| :----- | :----- | :----- |
+| 1.10.x | 31 Aug 2020 | 2 years |
+| 2.1.x | 4th Quarter 2020 | 2 years |
+
+Users of 1.9.3 or earlier are urged to upgrade to 1.10.0 and to consider the starting the migration 
+to prepare for the Accumulo 2.1.0 release.  
+
+## Notable Changes
+
+### Bug Fixes
+
+#### Addressed multiple possible concurrency issues
++ Prevent multiple threads from working on same bulk file. (Issue #1153)
```

Comment:
I am not sure these issue links will work as intended when rendered via Jekyll.  There is a [custom Jekyll plugin](https://github.com/apache/accumulo-website#custom-liquid-tags) that can do this using the syntax `{% ghi 1153 %}`.